src/main/java/frc/trigon/robot/subsystems/ampaligner/AmpAlignerConstants.java

```diff
@@ -0,0 +1,113 @@
+package frc.trigon.robot.subsystems.ampaligner;
+
+import com.ctre.phoenix6.configs.TalonFXConfiguration;
+import com.ctre.phoenix6.signals.*;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.system.plant.DCMotor;
+import edu.wpi.first.units.Units;
+import edu.wpi.first.wpilibj2.command.sysid.SysIdRoutine;
+import frc.trigon.robot.constants.RobotConstants;
+import org.trigon.hardware.RobotHardwareStats;
+import org.trigon.hardware.phoenix6.talonfx.TalonFXMotor;
+import org.trigon.hardware.phoenix6.talonfx.TalonFXSignal;
+import org.trigon.hardware.simulation.SingleJointedArmSimulation;
+
+public class AmpAlignerConstants {
+    private static final int MOTOR_ID = 13;
+    private static final String MOTOR_NAME = "AmpAlignerMotor";
+    static final TalonFXMotor MOTOR = new TalonFXMotor(MOTOR_ID, MOTOR_NAME, RobotConstants.CANIVORE_NAME);
+
+    private static final InvertedValue INVERTED_VALUE = InvertedValue.Clockwise_Positive;
+    private static final NeutralModeValue NEUTRAL_MODE_VALUE = NeutralModeValue.Brake;
+    private static final ReverseLimitTypeValue REVERSE_LIMIT_TYPE_VALUE = ReverseLimitTypeValue.NormallyOpen;
+    private static final ReverseLimitSourceValue REVERSE_LIMIT_SOURCE_VALUE = ReverseLimitSourceValue.LimitSwitchPin;
+    private static final double
+            P = RobotHardwareStats.isSimulation() ? 5 : 0,
+            I = RobotHardwareStats.isSimulation() ? 0 : 0,
+            D = RobotHardwareStats.isSimulation() ? 0 : 0,
+            KS = RobotHardwareStats.isSimulation() ? 0.13729 : 0,
+            KV = RobotHardwareStats.isSimulation() ? 5.7088 : 0,
+            KA = RobotHardwareStats.isSimulation() ? 0.13025 : 0;
+    static final double KG = RobotHardwareStats.isSimulation() ? 0.007923 : 0;
```

Comment:
Didn't we make two kGs so everything will operate correctly on sim (no weird kG stuff)